agreement/message_test.go

```diff
@@ -111,6 +112,9 @@ func TestMessageBackwardCompatibility(t *testing.T) {
 		Tag: protocol.ProposalPayloadTag,
 	}
 
+	if !bytes.Contains(encoded, []byte("MessageHandle")) {
+		require.FailNow(t, "encoded message does not contain MessageHandle field")
```

Comment:
[require.Containsf](https://pkg.go.dev/github.com/stretchr/testify/require#Containsf) ?

Reply:
Thanks yeah, I did it originally but it doesn't work with byteslices but just realized that it's the same if i cast the contains string to bytes or the encoded bytes to string. Thanks!